archlinuxcn/llvm-git/PKGBUILD

```diff
@@ -190,7 +191,7 @@ package_clang-git() {
 
     pkgdesc="C language family frontend for LLVM (git version)"
     url="http://clang.llvm.org/"
-    depends=("llvm-git=$pkgver-$pkgrel" 'gcc' "python" 'python2')
+    depends=("llvm-git=$pkgver-$pkgrel" 'gcc' "python" 'python2' 'perl')
```

Comment:
Optional? Do every command in this package uses it? (I kinda doubt `clang` do at the least). The official clang package does not seem to have `perl` as dependency or optional dependency. The python/python2 dependency should also probably be removed if not made optional.

Reply:
And looking at the optional dependency below, I think `llvm-libs-git=$pkgver-$pkgrel` should be a dependency whereas `llvm-git` should be an optional one....